### DISCUSSION TITLE
Does capacitor have gravitysensor/motionsensor function like from android studio docs?

### DISCUSSION THREAD

**Discussion by chiqors** [ionic-team/capacitor#5180] [Q&A] [answered]
i saw the documentation about motion sensor on capacitor doc (https://capacitorjs.com/docs/apis/motion). i couldn't see where i should begin with. I really want to try use capacitor for detecting x y z gravity.. they don't have good explanation regarding it. or maybe i am just too dumbfounded. Does web technology capable to get gravity sensor or motion sensor function from android studio doc? Thanks

Here's the android studio doc links that i want to discuss (https://developer.android.com/guide/topics/sensors/sensors_motion#sensors-motion-grav)

**Comment by jcesarmobile** [comment, marked as answer]
Motion plugin includes `acceleration` and `accelerationIncludingGravity`

> accelerationIncludingGravity:  An object giving the acceleration of the device on the three axis X, Y and Z with the effect of gravity. Acceleration is expressed in m/s


https://capacitorjs.com/docs/apis/motion#accellistenerevent

**Reply by chiqors** [reply]
would you like to describe, the function `accelerationIncludingGravity` of how it works in code? because, i had no idea.. sry

**Reply by jcesarmobile** [reply]
you can check our test app
https://github.com/ionic-team/capacitor-testapp/blob/main/src/pages/Motion.tsx#L45-L49

event will include both ` acceleration` and `accelerationIncludingGravity`

**Reply by chiqors** [reply]
it doesn't do anything from https://github.com/ionic-team/capacitor-testapp
i have pressed anything, idk why
![Screenshot_2021-10-29-18-39-04-021_com capacitorjs app testapp](https://user-images.githubusercontent.com/13832788/139428451-c698ffee-a514-45d3-92dc-da900cd977ce.jpg)

**Reply by jcesarmobile** [reply]
it should be showing the values in the console, you can check it in Android studio, search the logcat tab or in desktop chrome browser, enter this url chrome://inspect/#devices and your device should appear if you have enabled developer tools.

**Reply by chiqors** [reply]
oh, i didn't check the code first.. it was console log.. no wonder i couldn't see anything in front of app. Thank you so much
